### PR TITLE
Allen

### PR DESCRIPTION
**Pull request by Allengoyne**
sidebar content

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,10 +5,90 @@
...
+
+        <div id="price">
+          <p>Price</p>
+          <span></span>
+        </div>
+
+        <div id="filterByColor">
+          <p>Filter by Color</p>
+          <span></span>
+
```
Comment (jeremybdavis) [inline index.html]: Missing closing div tag

File: index.html [hunk 2]
```diff
@@ -5,10 +5,90 @@
...
+        </div>
+
+        <div id="price">
+          <p>Price</p>
+          <span></span>
+        </div>
+
+        <div id="filterByColor">
+          <p>Filter by Color</p>
+          <span></span>
```
Comment (jeremybdavis) [inline index.html]: let's remove the span for now

Comment (Allengoyne) [inline index.html]: ok

File: index.html (hunk 1 above)
Comment (Allengoyne) [inline index.html]: got it

**Pull request closed without merging**